### PR TITLE
Minutes tech lead 13 august 2024

### PR DESCRIPTION
**Pull request by valeriupredoi**
:duck: https://hackmd.io/VceghTxTT5-qgYMkV4pAyA?both

### CODE REVIEW

File: Tech Lead/Minutes/20240813.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- @schlunma: how can we support variable derivation for ERA5 in the fix? https://github.com/ESMValGroup/ESMValCore/issues/1806
+- @valeriupredoi: development dependencies test https://github.com/ESMValGroup/ESMValCore/issues/2492
+- @valeriupredoi: PRs that need ESMValCore a long time before release cycle https://github.com/ESMValGroup/ESMValTool/pull/3697
+- @valeriupredoi: ESMValTool ready for Python 3.12 bar `prospector` https://github.com/ESMValGroup/ESMValTool/pull/3501
+- @schlunma: pin iris? https://github.com/ESMValGroup/ESMValCore/pull/2500
+- @bouweandela: ESMValBot maintainance
+
+## Topics discussed
+
+- let's come back to how can we support variable derivation for ERA5 in the fix? https://github.com/ESMValGroup/ESMValCore/issues/1806
```
Comment (schlunma) [inline Tech Lead/Minutes/20240813.md:25]:
```suggestion
- Variable derivation for ERA5 in the fix (https://github.com/ESMValGroup/ESMValCore/issues/1806)? -> @bouweandela will at some point look into it
```

If I remember correctly?

File: Tech Lead/Minutes/20240813.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- ...we should have another look at the RTW PR in `main` https://github.com/ESMValGroup/ESMValTool/pull/3210
+- @valeriupredoi: ESMValTool ready for Python 3.12 bar `prospector` https://github.com/ESMValGroup/ESMValTool/pull/3501 - the only showstopper is, as we mentioned, `prospector` which, according to @bouweandela, and everyone else, is a bit dead; @valeriupredoi will start the integration process of Ruff into Codacy (since we really want Codacy umbrella) after holidays in September;
+- @schlunma: pin iris? https://github.com/ESMValGroup/ESMValCore/pull/2500 - that PR is now Ready-for_Review, it is great to have two supported Iris versions (latest, one before latest) but if API changes are large, we can go with just the latest on occasion (like now)
+- @bouweandela: ESMValBot maintainance: the Bot needs updating (really old deps), and needs migration to the new Levante tenancy; @valeriupredoi will help with this September onwards ->
```
Comment (schlunma) [inline Tech Lead/Minutes/20240813.md:31]:
```suggestion
- @bouweandela: ESMValBot maintainance: the Bot needs updating (really old deps), and needs migration to the new Levante tenancy; @valeriupredoi will help with this September onwards
```

**Review by schlunma — COMMENTED** [review]
Great! Just some tiny comments.

**Comment by valeriupredoi** [comment]
excellent, cheers, Manu :beer:

**Pull request merged**